lib/cc/services/github_pull_requests.rb

```diff
@@ -13,13 +13,35 @@ class Config < CC::Service::Config
       label: "Github Context",
       description: "The integration name next to the pull request status",
       default: "codeclimate"
+    attribute :welcome_comment_enabled, Axiom::Types::Boolean,
+      label: "Welcome comment enabled?",
+      description: "Should Code Climate post a welcome comment on pull requests?",
```

Comment:
:thumbsup: I'll make that change now and plan to share screenshots with copy folks when they're ready to get a coat of paint